hack/ci-e2e-kind-upgrade.sh

```diff
@@ -194,7 +188,13 @@ make test-pre-upgrade GARDENER_PREVIOUS_RELEASE=$GARDENER_PREVIOUS_RELEASE GARDE
 
 echo "Upgrading gardener version '$GARDENER_PREVIOUS_RELEASE' to '$GARDENER_NEXT_RELEASE'"
 upgrade_to_next_release
-wait_until_seed_gets_upgraded "$SEED_NAME"
+
+echo "Wait until seed '$SEED_NAME' gets upgraded from version '$GARDENER_PREVIOUS_RELEASE' to '$GARDENER_NEXT_RELEASE'"
+kubectl wait seed $SEED_NAME --timeout=5m --for=jsonpath="{.status.gardener.version}=$GARDENER_NEXT_RELEASE"
+# RETRY_LIMIT has been increased to 240 (20 minutes) due to the upgrading of Gardener for seed.
+# In a single-zone setup, 3 istio-ingressgateway pods will be running, and it will take 9 minutes to complete the rollout.
+# In a multi-zone setup, 6 istio-ingressgateway pods will be running, and it will take 18 minutes to complete the rollout.
```

Comment:
Where are these values coming from? 😕 Why does it take `9m`/`18m`? Can we speed this up?

Also, the upgrade tests still fail, why?

Reply:
> Where are these values coming from? 😕 Why does it take 9m/18m? Can we speed this up?

While upgrading Gardener, each `istio-ingressgateway` pod was healthy after 3 mins, so rollout to complete we need `9m`/ `18m` for `single/muti-zone` setups.
@ScheererJ  Is it possible to reduce time?

> Also, the upgrade tests still fail, why?
There is a downtime for while upgrading Gardener, I'm investigating more